src/modules/mc_att_control/mc_att_control_main.cpp

```diff
@@ -540,6 +559,14 @@ MulticopterAttitudeControl::parameters_update()
 	param_get(_params_handles.pitch_rate_ff, &v);
 	_params.rate_ff(1) = v;
 
+	param_get(_params_handles.d_term_cutoff_freq, &_params.d_term_cutoff_freq);
+	_lp_filters_d[0].set_cutoff_frequency(_loop_update_rate_hz, _params.d_term_cutoff_freq);
+	_lp_filters_d[1].set_cutoff_frequency(_loop_update_rate_hz, _params.d_term_cutoff_freq);
+	_lp_filters_d[2].set_cutoff_frequency(_loop_update_rate_hz, _params.d_term_cutoff_freq);
+	_lp_filters_d[0].reset(_rates_prev(0));
```

Comment:
Reset on any param change?

Reply:
Right, I changed it to update only when the frequency changes.